### ISSUE TITLE
Visualization fail

### ISSUE THREAD

**Issue by jiaxiangshang** [abdallahdib/NextFace#27]
Hi, abdallahdib,
1. I run your code, the visualization of envMaps is black,
![envMap_0](https://user-images.githubusercontent.com/21188156/173610227-52700227-4002-4a6b-b3db-886b8d801b56.png)

3. the visualization of  
`
  renderAll = torch.cat([
      torch.cat([inputTensor[i], torch.ones_like(images[i])[..., 3:]], dim = -1),
      torch.cat([overlay.to(self.device), torch.ones_like(images[i])[..., 3:]], dim = -1),
      images[i],
      #illum[i],
      diffuseAlbedo[self.getTextureIndex(i)],
      specularAlbedo[self.getTextureIndex(i)],
      roughnessAlbedo[self.getTextureIndex(i)]], dim=1)
`
is transparent as below.
![render_0](https://user-images.githubusercontent.com/21188156/173611380-88ebccf5-f232-40ab-b442-8abfbd36f933.png)

The loss figure is here,
![stage1_loss](https://user-images.githubusercontent.com/21188156/173610719-8c46a45d-a166-48a5-b03c-2a38e45d08b4.png)

Do you have further suggestions, the overlay rendering results seems right.

**Comment by abdallahdib** [comment]
Hi, 
this is weird can u plz share the image so that i can try and debug it

**Comment by abdallahdib** [comment]
plz share ur config file u r using for the optim as well 
Tx

**Comment by jiaxiangshang** [comment]
Hi abdallahdib

Thank you for your quick reply, I have not modified the config, but I change some IO code to load the input image.
I will try your origin code first, if there still some problems, I will send the images to you,

Thanks again for your kind help,

Best
Jiaxiang
